src/components/AFEpage/index.js

```diff
@@ -1,85 +1,199 @@
 import React from "react";
-import "./styles.scss";
+import useStyles from "./styles";
+import { breakpoints } from "../../theme/constant";
+import {
+  Container,
+  Typography,
+  Grid,
+  useMediaQuery,
+  List,
+  ListItem,
+  ListItemText,
+} from "@mui/material";
+import FiberManualRecordIcon from "@mui/icons-material/FiberManualRecord";
 
 function AFEpage() {
+  const classes = useStyles();
+  const isActive = useMediaQuery("(max-width:" + breakpoints.values.sm + "px)");
```

Comment:
However, `isActive` is actually not necessary here as you can use [responsive values](https://mui.com/components/grid/#responsive-values) in each case except possibly for the abbreviation, for which I suggest a new component.  See the suggestions below.